Quick-Start [DA]/MVP/MVP+Makefile.md

```diff
@@ -399,4 +398,201 @@ To set up and use PGXN to publish and distribute your extension to other users,
    Notes on upgrading to a new version and any migration steps required.
 ```
 
-5.  **Publish Your Extension:** Use the PGXN command-line tool or web interface to publish your extension. Provide the distribution package created in the previous step and upload it to PGXN. The tool will validate the package and extract the metadata. 
+4.  **Build a Distribution Package:** Package your extension and the `META.json` file into a distribution package. This package should be in a compressed format (zip format) and include all the necessary files for installation and usage of your extension.
+
+5.  **Publish Your Extension:** Use the PGXN command-line tool or web interface to publish your extension. Provide the distribution package created in the previous step and upload it to PGXN. The tool will validate the package and extract the metadata.
+
+### Updating an Extension on PGXN:
+#### New Directory Hierarchy:
+```
+my_extension/
+├── changes
+├── META.json
+├── README.md
+├── my_extension--1.0.0.sql
+├── my_extension--1.0.1.sql
+├── expected/
+│   └── my_extension--regress.out
+├── sql/
+│   └── my_extension--regress.sql
+├── my_extension.control
+├── results/
+│   └── my_extension--regress.out
+└── Makefile
+```
+
+#### Step 1: Update the Extension Control File:
+- Open the "my_extension.control" file and update the following content:
+
+```control
+# my_extension.control
+comment = 'Minimal Viable Product'
+default_version = '1.0.1'
+relocatable = true
+```
+
+#### Step 2: Create the new Extension SQL File:
+
+- Create a file named `my_extension--1.0.1.sql`.
+
+  - This file contains the new SQL statements for the extension.
+  - The format for a PostgreSQL extension file name is `extension_name--version.sql`, where `extension_name` is the name of the extension, and `version` is the version number. The double-dash serves as a delimiter to indicate the separation between the extension name and the version number.
+
+```bash
+$ touch my_extension--1.0.1.sql
+```
+
+- Open the `my_extension--1.0.1.sql` file and add the following content:
+  - The `complex_add` is a new function named complex_add that takes an array of integers as input. Inside the function, we use the unnest function to expand the array into individual elements and then calculate the sum using the SUM function. The result is returned as an integer.
```

Comment:
Expanding the extension with new functionality is a great lesson, but ought to be independent of distribution on PGXN, I think.

Reply:
What i'll do is i will keep the new function but remove all the information about the PGXN. I will cover that in a separate lesson.(I can give some information about PGXS too as it isn't too long)